### ISSUE TITLE
uefi.ntfs.img appears to be larger than 512kB recently

### ISSUE THREAD

**Issue by arjenv** [WoeUSB/WoeUSB#81]
## Describe the bug ##

Dear developers. I am trying to create a Windows10 USB bootable disk with woeusb. After some initial hickups I could solve, I ended up with an error stating that UEFI-NTFS.IMG is too large. I posted this question in the ask-ubuntu forum and the developer from uefi.ntfs answered that he raised the size from 512kB to 1MB just recently. He suggested contacting you with this issue.

## To Reproduce ##

sudo woeusb -d /home/xxx/Downloads/Win10_21H1_EnglishInternational_x64.iso /dev/sdc --tgt-fs NTFS --no-color

WoeUSB v3.3.1

## Teminal output ##

If applicable, add terminal output to help explain your problem.  Please avoid pasting screenshots whenever possible.

```
Mounting source filesystem...
Wiping all existing partition table and filesystem signatures in /dev/sdc...
/dev/sdc: 2 bytes were erased at offset 0x000001fe (dos): 55 aa
/dev/sdc: calling ioctl to re-read partition table: Success
Ensure that /dev/sdc is really wiped...
Creating new partition table on /dev/sdc...
Creating target partition...
Making system realize that partition table has changed...
Wait 3 seconds for block device nodes to populate...
Cluster size has been automatically set to 4096 bytes.
Creating NTFS volume structures.
mkntfs completed successfully. Have a nice day.
--2021-10-26 00:13:33--  https://github.com/pbatard/rufus/raw/master/res/uefi/uefi-ntfs.img
Resolving github.com (github.com)... 140.82.121.3
Connecting to github.com (github.com)|140.82.121.3|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: https://raw.githubusercontent.com/pbatard/rufus/master/res/uefi/uefi-ntfs.img [following]
--2021-10-26 00:13:33--  https://raw.githubusercontent.com/pbatard/rufus/master/res/uefi/uefi-ntfs.img
Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.110.133, 185.199.111.133, 185.199.109.133, ...
Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.110.133|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 1048576 (1,0M) [application/octet-stream]
Saving to: ‘/tmp/WoeUSB.CsW6q3.tempdir/uefi-ntfs.img’

uefi-ntfs.img       100%[===================>]   1,00M  --.-KB/s    in 0,1s    

2021-10-26 00:13:33 (8,75 MB/s) - ‘/tmp/WoeUSB.CsW6q3.tempdir/uefi-ntfs.img’ saved [1048576/1048576]

dd: writing to '/dev/sdc2': No space left on device
1025+0 records in
1024+0 records out
524288 bytes (524 kB, 512 KiB) copied, 0,589478 s, 889 kB/s
The command "dd if="${download_directory}/uefi-ntfs.img" of="${uefi_ntfs_partition}"" failed with exit status "1", program is prematurely aborted
Unmounting and removing "/media/woeusb_source_1635200001_9737"...
You may now safely detach the target device
```

## Environment ##

- OS: tried both Fedora 34 as Ubuntu 18.04
- Bash: not sure
- WoeUSB: [v3.3.1]

## Additional context ##

See:
https://askubuntu.com/questions/1371599/trying-to-create-bootable-windows10-usb-drive-with-woeusb-seems-uefi-ntfs-img-i

**MAINTAINER EDIT:** Fix code block markup

**Comment by brlin-tw** [comment]
Apologies for the ignorance, I've merged the pull request that contains the fix and will release it shortly after testing.

**Comment by brlin-tw** [comment]
The bug is fixed in [WoeUSB 5.1.3](https://github.com/WoeUSB/WoeUSB/releases/tag/v5.1.3).

**Comment by arjenv** [comment]
Thanks! Much appreciated!
